index.js

```diff
@@ -942,7 +977,7 @@
   //. > S.init([])
   //. Nothing()
   //. ```
-  S.init = R.ifElse(R.isEmpty, Nothing, R.compose(Just, R.init));
+  S.init = S.slice(0, -1);
```

Comment:
The implementation is already in there. :) (Although I had forgotten to add a test for the -0 functionality in the `S.at` tests.)